Solutions/Threat Intelligence/Analytic Rules/DomainEntity_CommonSecurityLog.yaml

```diff
@@ -1,7 +1,7 @@
 id: dd0a6029-ecef-4507-89c4-fc355ac52111
 name: TI map Domain entity to CommonSecurityLog
 description: |
-  'Identifies a match in CommonSecurityLog table from any Domain IOC from TI'
```

Comment:
As the PR description, description is improved and external IOCs are removed. I don't see those changes here.
Only minor typo mistakes are fixed. Is this the intended PR you want to submit?